contracts/examples/SimpleInterestTermsContract.sol

```diff
@@ -271,6 +297,17 @@ contract SimpleInterestTermsContract is TermsContract {
         return delta.div(amortizationUnitLengthInSeconds);
     }
 
+    function calculatePrincipalPlusInterest(
+        SimpleInterestParams params
+    )
+        internal
+        returns (uint _principalPlusInterest)
+    {
+        uint interestPayment = params.principalAmount.mul(params.interestRate);
+        uint totalInterest = interestPayment.mul(params.termLengthInAmortizationUnits);
```

Comment:
let's add a comment to this function explaining that we define interest as accruing once per amortization period.

Additionally, let's change the function name to `calculateTotalPrincipalPlusInterest` for clarity -- given that this is the total expected repayment value.